conans/test/unittests/client/rest/uploader_test.py

```diff
@@ -44,13 +48,15 @@ def put(self, *args, **kwargs):
     def test_403_raises_forbidden_exception_if_token(self):
 
         class MockRequester(object):
+            retry = 0
```

Comment:
Is this needed in the current implementation? Are the retry finally members of the requester?